api/middlewares/urlsMiddlewares.js

```diff
@@ -0,0 +1,24 @@
+const { StatusCodes } = require("http-status-codes");
+
+const handleNoUrlsFoundError = (error, req, res, next) => {
+  const message =
```

Comment:
checking if the status code is equal to StatusCodes.NOT_FOUND prevent the function to be generic.
u can check if error.statusCode !== null (because only the custom error have the property of statusCode).